src/api/app/assets/stylesheets/webui2/build-results.scss

```diff
@@ -1,6 +1,7 @@
 .rpmlint-result {
-  font-size: .7rem;
+  font-size: .9em;
   max-height: 35rem;
+  font-family: $font-family-sans-serif;
```

Comment:
in that case, because we want to simulate the output of a shell, IMO it would be ok to have a different font stack like:  

`font-family: Inconsolata, Consolas, monospace`

What do you think @hennevogel ? Do you want to have it look like a shell output?

Reply:
however the original font-stack, `font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace; ` is as well good!

Reply:
Hm, I'm not sure that we want to simulate output of a shell. You are right, it's kind of a log. Maybe we need a component for things like this?